src/core/i18n/utils.js

```diff
@@ -230,6 +232,39 @@ function oneLineTranslationString(translationKey) {
   return translationKey;
 }
 
+type FormatFilesizeParams = {|
+  _filesize: typeof filesize,
+  i18n: I18nType,
+  size: number,
+|};
+
+// Translates a file size in bytes into a localized user-friendly format.
+export const formatFilesize = ({
+  _filesize = filesize,
+  i18n,
+  size,
+}: FormatFilesizeParams): string | null => {
+  const sizeStrings = {
+    // These are the expected values for the unit of measure returned by
+    // filesize. Realistically we shouldn't get anything back larger than TB.
+    B: i18n.gettext('%(formattedSize)s B'),
```

Comment:
Calling this `%(localizedSize)s` may give translators a better hint about what's happening.